src/translations/views/TranslationsEntities.tsx

```diff
@@ -176,7 +176,8 @@ const TranslationsEntities: React.FC<TranslationsEntitiesProps> = ({
           }}
         </TypedCategoryTranslations>
       ) : params.tab === "products" ? (
-        <TypedProductTranslations variables={queryVariables}>
+        // FIXME: Remove hardcoded variable
+        <TypedProductTranslations variables={{ ...queryVariables, first: 5 }}>
```

Comment:
So should this be fixed now or why is this so?

Reply:
This should be fixed now on backend and should be ready to remove. @kamilpastuszka please confirm

Reply:
Confirmed, that's been fixed. I’ve removed to comment and changed the limit to the previous value of 20.